config/labels.yml

```diff
@@ -177,6 +177,11 @@ repos:
     <<: *feature_introduced_in_darga_or_prior
     <<: *provider_plugin
     <<: *release
+  manageiq-providers-kubevirt:
+    <<: *common
+    <<: *feature_introduced_in_gaprindashvili
```

Comment:
I think it's only in hammer, so this should just be commented out with `feature_introduced_in_hammer` like this one: https://github.com/ManageIQ/manageiq-release/pull/56/files#diff-a5375cf6511f0bf938d99ae477700393R139